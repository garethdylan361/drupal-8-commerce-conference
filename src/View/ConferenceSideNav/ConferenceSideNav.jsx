class ConferenceSideNav extends React.Component {

  constructor(props){
    super(props);
  }

  handleSelect(selectedKey) {
    event.preventDefault();
  }

  render(){

    let Nav = ReactBootstrap.Nav,
      NavItem = ReactBootstrap.NavItem;

    return(
      <Nav bsStyle="pills" stacked activeKey={1} onSelect={this.handleSelect}>
        <NavItem eventKey={1}>Conference 2017</NavItem>
        <NavItem eventKey={1}>Conference 2016</NavItem>
        <NavItem eventKey={1}>Conference 2015</NavItem>
        <NavItem eventKey={1}>Conference 2014</NavItem>
        <NavItem eventKey={1}>Conference 2013</NavItem>
        <NavItem eventKey={1}>Conference 2012</NavItem>
        <NavItem eventKey={1}>Conference 2011</NavItem>
      </Nav>
    );
  }
}
