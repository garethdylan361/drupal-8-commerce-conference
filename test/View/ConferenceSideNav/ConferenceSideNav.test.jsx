import React from 'react';
import renderer from 'react-test-renderer';
import ConferenceSideNav from '../../../src/View/ConferenceSideNav/ConferenceSideNav';

test('ConferenceSideNav renders correctly', () => {
  const sideNav = renderer.create(<ConferenceSideNav/>);
  expect(sideNav).toMatchSnapshot();
});
