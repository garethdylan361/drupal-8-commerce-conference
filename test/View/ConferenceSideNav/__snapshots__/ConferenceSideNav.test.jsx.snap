// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConferenceSideNav renders correctly 1`] = `
<ul
  className="nav nav-pills nav-stacked"
  role={null}
>
  <li
    className="active"
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Conference 2017
    </a>
  </li>
  <li
    className=""
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Conference 2016
    </a>
  </li>
  <li
    className=""
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Conference 2015
    </a>
  </li>
  <li
    className=""
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Conference 2014
    </a>
  </li>
  <li
    className=""
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Conference 2013
    </a>
  </li>
  <li
    className=""
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Conference 2012
    </a>
  </li>
  <li
    className=""
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Conference 2011
    </a>
  </li>
</ul>
`;
